aiida/engine/daemon/client.py

```diff
@@ -334,27 +340,28 @@ def get_tcp_endpoint(self, port=None):
 
     @property
     def client(self) -> 'CircusClient':
-        """
-        Return an instance of the CircusClient with the endpoint defined by the controller endpoint, which
-        used the port that was written to the port file upon starting of the daemon
+        """Return an instance of the CircusClient.
 
-        N.B. This is quite slow the first time it is run due to the import of zmq.ssh
-        in circus/utils.py in circus 0.15.0, which ultimately follows the import of CircusClient.
+        The endpoint is defined by the controller endpoint, which used the port that was written to the port file upon
+        starting of the daemon.
+
+        .. note:: This is quite slow the first time it is run due to the import of ``zmq.ssh`` in ``circus/utils.py`` in
```

Comment:
Do we make use of ``zmq.ssh`` or is this import effectively superfluous for us?

Reply:
Thanks. Yes, no need to fix it in this PR.
If you believe it may be a valid candidate to things up (e.g. because we don't use the `ssh` thing), perhaps worth mentioning it here, that's all.

Reply:
I moved the import top-level and it actually caused the verdi load time test to fail. Now this could have been coincidence, but best to keep the import in the method. No real reason not to.